libs/blocks/merch-card/merch-card.css

```
@@ -103,6 +111,16 @@
   border-radius: 16px;
 }
 
+.merch-card.special-offers.full-width {
+  min-width: 100%;
+  max-width: 100%;
+  width: var(--grid-container-width);
+}
+
+.merch-card.special-offers.center {
```

```suggestion
.merch-card.center {
```
if we want this on other card types? 🤷

---

Any author-able alignment considerations for the cta row? if we we might need a few more flex align props.

---

-Only Special Offers Card was requested to have this property. 
-CTA row alignment is not in scope as per Consonant XD style guide so far. We can always introduce the change if there is a requirement.